debian/marquee/nginx/sites/domparsing.spec.whatwg.org.conf

```diff
@@ -0,0 +1,13 @@
+server {
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
+
+    server_name domparsing.spec.whatwg.org;
+
+    ssl_certificate /etc/letsencrypt/live/domparsing.spec.whatwg.org/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/domparsing.spec.whatwg.org/privkey.pem;
+
+    include /etc/nginx/whatwg.conf;
+
+    return 301 https://w3c.github.io/DOM-Parsing$request_uri;
```

Comment:
Hmm, this matches the apache config, but maybe we should omit `$request_uri`, as basically nothing in https://github.com/whatwg/domparsing exists in https://github.com/w3c/DOM-Parsing